.bluemix/nls/messages.yml

```diff
@@ -7,6 +7,7 @@ deploy.description: "sample toolchain"
 deploy.longDescription: "The Delivery Pipeline automates continuous deployment."
 deploy.appDescription: "The name of your Watson app"
 deploy.appName: "App name"
+deploy.metricsDescription : "The metrics tracker is for IBM to track metrics around deployments of sample applications to IBM Bluemix to measure the usefulness of our examples, so that we can continuously improve the content we offer to you. If you want to disable the metrics tracker, please put down YES."

```

Comment:
We should state clearly the default behavior. Please change this to:

"The metrics tracker, which is enabled by default, is for IBM to track the number of times our examples are deployed to IBM Cloud. It is strictly used to measure the usefulness of our examples so that we can continuously improve the content we offer to you. If you want to disable the metrics tracker, please put down YES."